test/OrchardCore.Tests/Modules/OrchardCore.Security/Extensions/SecurityHeadersBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using Xunit;
+
+namespace OrchardCore.Security.Extensions.Tests
+{
+    public class SecurityHeadersBuilderExtensionsTests
+    {
+        [Fact]
+        public void AddContentTypeOptionsWithFluentAPIsConfiguration()
```

Comment:
Similarly with the others too.

```suggestion
        public void AddContentTypeOptionsWithFluentApisConfiguration()
```